packages/flutter_tools/lib/src/commands/update_packages.dart

```diff
@@ -39,12 +39,14 @@ class UpdatePackagesCommand extends FlutterCommand {
         help: 'Attempt to update all the dependencies to their latest versions.\n'
```

Comment:
all actual changes are in this file

Reply:
Originally were a workaround for dart 1 and dart 2 support, can probably be removed now